magento/magento2
# [MFTF] Refactoring AdminSortingByWebsitesTest

### Description (*)
The test is refactored according to the best practices followed by MFTF.

### Manual testing scenarios (*)
1. Create a custom website
2. Assign the created website to a product
3. Go to Catalog->Products
4. Sort the Products grid by the "Website" column asc
5. Check if the product in the first row is assigned to Main Website
6. Sort the Products grid by the "Website" column desc
7.  Check if the product in the first row the created custom website


### Resolved issues:
1. [x] resolves magento/magento2#31442: [MFTF] Refactoring AdminSortingByWebsitesTest
